resources/containers/field-modal.tsx

```diff
@@ -19,35 +19,37 @@ const FieldModal: React.FC<FieldModalProps> = ({
 }) => {
   const [selectedOptions, setSelectedOptions] = useState<string[]>([]);
   const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const [newOption, setNewOption] = useState<string>("");
 
-  // Convert enum to array of values and omit the fieldType
-  const options = Object.values(TYPE_OPTIONS).filter(
+  const typeOptions = Object.values(TYPE_OPTIONS).filter(
     (option) => option !== fieldType,
   );
 
-  useEffect(() => {
-    setSelectedOptions([]);
-  }, [isOpen]);
+  const handleSave = () => {
+    if (fieldType === TYPE_OPTIONS.NUMBER && selectedOptions.length > 0) {
+      onSave({ maxValue: parseInt(selectedOptions[0]) });
+    } else if (
+      [TYPE_OPTIONS.SELECT, TYPE_OPTIONS.MULTISELECT].includes(
+        fieldType as TYPE_OPTIONS,
+      )
+    ) {
+      onSave({ options: selectedOptions });
```

Comment:
Dont need if else, we just save the outpout of the form with react-foorm, whatever form type it is. 
The form already knows how to validate it self.

Reply:
If you don't undertand my comnets, create a new task on the board to do next week, if this stable I can accept for now